packages/security/src/deviceTrust/SecurityCheck.ts

```diff
@@ -1,7 +1,18 @@
 import { SecurityCheckResult } from "./SecurityCheckResult";
 
+/**
+ * Interface for a device security single check that can be executed.
+ */
 export interface SecurityCheck {
+  /**
+   * The name of the check. This can be used for reporting the checks results.
+   */
   name: string;
 
+  /**
+   * Execute the check.
+   *
+   * @returns {Promise<SecurityCheckResult>}
```

Comment:
TSdocs actually generates this types so this is redundant. We can just say in normal words what is returned.